flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/service/AbstractFlinkService.java

```diff
@@ -594,6 +638,58 @@ public SavepointFetchResult fetchSavepointInfo(
         }
     }
 
+    @Override
+    public CheckpointFetchResult fetchCheckpointInfo(
```

Comment:
Would be great to have a test for this in the AbstractFlinkServiceTest similar to the triggering test

Reply:
Oh, missed that one. Added.